app/Http/Requests/UpdateManagerProfileRequest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use App\Services\Validation\Rules\ValidIdRule;
+use App\Models\Lookup\Frequency;
+use App\Models\Lookup\Department;
+use App\Services\Validation\Rules\LinkedInUrlRule;
+use Illuminate\Validation\Rule;
+use App\Services\Validation\Rules\PasswordCorrectRule;
+use App\Services\Validation\Rules\PasswordFormatRule;
+use App\Services\Validation\Rules\TwitterHandleRule;
+
+class UpdateManagerProfileRequest extends FormRequest
+{
+    /**
+     * Validator instance updated on failedValidation.
+     *
+     * @var \Illuminate\Contracts\Validation\Validator
+     */
+    public $validator = null;
+
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return boolean
+     */
+    public function authorize()
+    {
+        $manager = $this->route('manager');
+        return $manager->id !== null;
+    }
+
+    /**
+     * Override Handle a failed validation attempt.
+     *
+     * @return void
+     */
+    protected function failedValidation(\Illuminate\Contracts\Validation\Validator $validator)
+    {
+        $this->validator = $validator;
```

Comment:
What's the purpose of overriding this?

Reply:
This is so I have access to the `$request->validator->fails()` method. With that I can scroll to errors section. 

```php
// redirect to error messages element if validation fails
        if (isset($request->validator) && $request->validator->fails()) {
            $hash = '#managerProfileFormErrors';
            return redirect(route('manager.profile.edit', $manager).$hash)
                        ->withErrors($request->validator)
                        ->withInput();
        }
```